static/js/server_events.js

```diff
@@ -91,8 +91,11 @@ function dispatch_normal_event(event) {
         } else if (event.op === 'remove') {
             bot_data.remove(event.bot.email);
         } else if (event.op === 'update') {
+            if (_.has(event.bot, 'owner_id')) {
+                event.bot.owner = people.get_person_from_user_id(event.bot.owner_id).email;
```

Comment:
`bot_data.js` will not pick up the changes. We don't have a `owner_id` field in `bot_data.js`. We are storing owner's email only there.